src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -77,6 +79,13 @@ public Command parseCommand(String userInput) throws ParseException {
         case HelpCommand.COMMAND_WORD:
             return new HelpCommand();
 
+        case CalculateCAPCommand.COMMAND_WORD:
```

Comment:
I think the value will end up being "calculate" because the regex matches the value before the space.

Reply:
Fix according to comments👍